client/src/components/CreatePetForm.jsx

```diff
@@ -168,34 +175,29 @@ export function CreatePetForm() {
                 </Option>
               </Select>
               {/* Breed dropdown */}
-              <Select
-                label="Breed"
-                id="breed"
-                value={breed}
-                onChange={onBreedChange}
-                disabled={!species}
-                required
-              >
-                {species == "cat"
-                  ? catBreedList.map((breed) => (
-                      <Option
-                        key={breed.id}
-                        name={breed.name}
-                        value={breed.name}
-                      >
-                        {breed.name}
-                      </Option>
-                    ))
-                  : dogBreedList.map((breed) => (
-                      <Option
-                        key={breed.id}
-                        name={breed.name}
-                        value={breed.name}
-                      >
-                        {breed.name}
-                      </Option>
-                    ))}
-              </Select>
+              {species == "cat" ? (
+                <Select
+                  label="Breed"
+                  id="breed"
+                  value={breed}
+                  onChange={onBreedChange}
+                  disabled={!species}
+                  required
+                >
+                  {catBreedListOptions}
```

Comment:
Look at what's different in these two `if`s-- or any two ifs. Then try surrounding just the differences with the if. This ends up DRYing up the code... int his case, I think you can just have a `{species == 'cat' ? catBreedListOptions : dogBreedListOptions }`... or similar.

Reply:
I think the bug fix I just pushed is, if possible, even more convoluted than what I had here...it works, but I am unsure how to clean it up without breaking selection again.